test/tools/ossfuzz/config/strict_assembly.dict

```diff
@@ -0,0 +1,118 @@
+" -> "
+" := "
+"("
+")"
+", "
+":"
+" dup1 "
+" dup10 "
+" dup11 "
+" dup12 "
+" dup13 "
+" dup14 "
+" dup15 "
+" dup16 "
+" dup2 "
+" dup3 "
+" dup4 "
+" dup5 "
+" dup6 "
+" dup7 "
+" dup8 "
+" dup9 "
+" pc "
+" stop "
+" swap1 "
+" swap10 "
+" swap11 "
+" swap12 "
+" swap13 "
+" swap14 "
+" swap15 "
+" swap16 "
+" swap2 "
+" swap3 "
+" swap4 "
+" swap5 "
+" swap6 "
+" swap7 "
+" swap8 "
+" swap9 "
+"add("
+"addmod("
+" address "
+"and("
+"balance("
+"blockhash("
+"byte("
+"call("
+"callcode("
+"calldatacopy("
+"calldataload("
+" calldatasize "
+" caller "
+" callvalue "
+"codecopy("
+" codesize "
+" coinbase "
+"create("
+"create2("
+"delegatecall("
+" difficulty "
+"div("
+"eq("
+"exp("
+"extcodecopy("
+"extcodehash("
+"extcodesize("
+"for "
+"function "
+" gas "
+" gaslimit "
+" gasprice "
+"gt("
+" invalid "
+"iszero("
+"jump("
+"jumpi("
+"keccak256("
+"log0("
+"log1("
+"log2("
+"log3("
+"log4("
+"lt("
+"mload("
+"mod("
+"msize"
+"mstore("
+"mstore8("
+"mul("
+"mulmod("
+"not("
+" number "
+"or("
+" origin "
+"pop("
+"return("
+"returndatacopy("
+" returndatasize "
+"revert("
+"sar("
+"sdiv("
+"selfdestruct("
+"sgt("
+"shl("
+"shr("
+"signextend("
+"sload("
+"slt("
+"smod("
+"sstore("
+"staticcall("
+"sub("
```

Comment:
`switch`, `case`, `default` and literal numbers (some examples) and strings are missing.

Reply:
done.